README.md

```diff
@@ -497,6 +497,90 @@ if (html instanceof Promise) {
 
 <br />
 
+### Why there is no `context` API?
+
+We choose to not provide a `context` API. Here the reasons why:
+
+This library only outputs strings (or Promises that resolve to strings), we don't manage
+lifecycle or state. So the main selling point to having a `context` API is to avoid prop
+drilling.
+
+A context Api would need to use a request identifier (that we are going to call
```

Comment:
the closest we can keep the documentation to the actual source code, easier it is to new maintainers help us out.

```suggestion
A context Api would need to use a request identifier (internally called `rid`)
to be safe to access asynchronously. Take for example two requests that are
```